gapis/api/vulkan/api/structs.api

```diff
@@ -926,8 +926,8 @@ class VkAttachmentDescription {
 }
 
 class VkAttachmentReference {
-  u32           Attachment // We have capitalized these so we can manipulate
-  VkImageLayout Layout     // them in Go
+  u32           attachment
```

Comment:
This was needed because we needed to access the fields. (In Go's `Attachment` is public, `attachment` is private). As we don't need to access it outside the package, we do not need to have them public anymore.